guardian/editorial-production-metrics
# Bump to Scala 2.13, SBT 1.8.2 and dependency updates

## What does this change?
This bumps Scala to 2.13, SBT to 1.8 and Play to 2.8 and updates all other dependencies to the latest stable version wherever possible.

## How to test
Run this locally or in CODE. Does everything in the UI still work as expected? If you want to test this in parallel with the production-metrics-lambda which is what feeds metrics to this application's database, you can follow the how to test instructions in this pr - https://github.com/guardian/production-metrics-lambdas/pull/14
